third_party/libpg_query/grammar/statements/create_alias.y

```diff
@@ -0,0 +1,14 @@
+/*****************************************************************************
+ *
+ * Create Alias Statement
+ *
+ *****************************************************************************/
+CreateAliasStmt:
+                CREATE_P TYPE_P any_name FROM Typename
```

Comment:
Can we unify this with the `CreateEnumStmt` (and maybe rename to `CreateTypeStmt`) rather than create a separate statement?